Gems/Atom/RHI/Vulkan/Code/Source/RHI.Reflect/Formats.inl

```diff
@@ -99,10 +99,6 @@
         _Func(ETC2A1_UNORM_SRGB,       VK_FORMAT_ETC2_R8G8B8A1_SRGB_BLOCK,      1,    0,    0) \
         _Func(ETC2A_UNORM,             VK_FORMAT_ETC2_R8G8B8A8_UNORM_BLOCK,     1,    0,    0) \
         _Func(ETC2A_UNORM_SRGB,        VK_FORMAT_ETC2_R8G8B8A8_SRGB_BLOCK,      1,    0,    0) \
-        _Func(PVRTC2_UNORM,            VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG,   1,    0,    0) \
-        _Func(PVRTC2_UNORM_SRGB,       VK_FORMAT_PVRTC2_2BPP_SRGB_BLOCK_IMG,    1,    0,    0) \
-        _Func(PVRTC4_UNORM,            VK_FORMAT_PVRTC2_4BPP_UNORM_BLOCK_IMG,   1,    0,    0) \
-        _Func(PVRTC4_UNORM_SRGB,       VK_FORMAT_PVRTC2_4BPP_SRGB_BLOCK_IMG,    1,    0,    0) \
```

Comment:
It might be a good idea to leave a comment in here indicating PVRTC formats are deprecated and have been removed because they cause validation errors in case some developer tries to add them back

Reply:
We shouldn't need to. There is no PVRTC format support in image builder.